src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2698,4 +2698,47 @@ limitations under the License.
             <include cluster="Device Energy Management Mode" client="false" server="true" clientLocked="true" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-waterheater</name>
+        <domain>CHIP</domain>
+        <typeName>Water Heater</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x050F</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <clusters>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>CLIENT_LIST</requireAttribute>
```

Comment:
Why all the requireAttribute for attributes which are mandatory on that cluster anyway?

Reply:
I checked with Hasty and he mentioned he just replicated the pattern in this file. It looks like in the past, we used to include all of these attributes. Over time, when manually adding entries, people stopped including these attributes but the tool still generates those.

Reply:
I suggest we leave this as-is for now as this is auto-generated.
If we want to remove those in the future we should probably update Alchemy directly and then regenerate the whole file.
FYI @hasty